distribution/api-docs-dist/pom.xml

```
@@ -34,6 +34,24 @@
         <maven.compiler.release>17</maven.compiler.release>
     </properties>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>com.redhat.quarkus.platform</groupId>
```

The `com.redhat` group ID is not available upstream. To be able to build it in upstream, I changed it to `io.quarkus`. Would that still work in downstream?

```suggestion
                <groupId>io.quarkus.platform</groupId>
```

---

Yes, you  are right. I am sorry for it.